### DISCUSSION TITLE
v4.0+ does not work on iOS/Safari 12.1.4

### DISCUSSION THREAD

**Discussion by HoneyBarbecue** [szimek/signature_pad#655] [Q&A]
v4.0+ does not work on iOS/Safari 12.1.4, v2.0+ works but couldn't use addEventListener method

Uploading 82707f508c34e55ad0043f38f05832e3.mp4…

**Comment by UziTech** [comment]
see #568. Comment there if you are having issues with ios. I don't have a mac or iphone so someone else will need to figure out what is wrong.
